tools/perf/Makefile

```diff
@@ -0,0 +1,131 @@
+TOP := $(shell pwd)
+TOOLS_DIR = $(HOME)/bin
+CURL_PATH ?= curl
+CURL_DOWNLOAD := $(CURL_PATH) --location --fail --progress-bar
+
+KIND_CLUSTER_NAME ?= kuma
+KIND_KUBECONFIG_DIR := $(HOME)/.kube
+KIND_KUBECONFIG := $(KIND_KUBECONFIG_DIR)/kind-$(KIND_CLUSTER_NAME)-config
+HELM_PATH := $(TOOLS_DIR)/helm
+KIND_PATH := $(TOOLS_DIR)/kind
+KUBECTL_PATH := $(TOOLS_DIR)/kubectl
+HELM_VERSION ?= v3.8.2
+KIND_VERSION ?= v0.14.0
+KUBECTL_VERSION ?= v1.23.5
+KUMA_VERSION ?= 1.8.0
+
+# wrk test configuration
+WRK_CONN ?= 100
+WRK_THREAD ?= 12
+WRK_DURATION ?= 60
+export PATH := $(TOOLS_DIR):$(PATH)
+CONFIG_PATH := single-node
+ifdef USE_MULTI_NODE
+	CONFIG_PATH = multi-node
+endif
+
+UNAME_OS := $(shell uname -s)
+UNAME_ARCH := $(shell uname -m)
+ARCH ?= UNAME_ARCH
+ifeq ($(UNAME_ARCH),x86_64)
+	ARCH = amd64
+endif
+
+OS ?= UNAME_OS
+ifeq ($(UNAME_OS),Linux)
+	OS = linux
+else ifeq ($(UNAME_OS),Darwin)
+	OS = darwin
+endif
+
+.PHONY: install/all
+install/all: install/kind install/helm install/kubectl
+
+.PHONY: install/kind
+install/kind: ## Bootstrap: Install KIND (Kubernetes in Docker)
+	# see https://kind.sigs.k8s.io/docs/user/quick-start/#installation
+	@if [ -e $(KIND_PATH) ]; then echo "Kind $$( $(KIND_PATH) version ) is already installed at $(KIND_PATH)" ; fi
```

Comment:
just_comment: you could have a real target based on a file, if the file is not present then `make` would execute the target.